Assets/Editor/ProdBuildScript.cs

```diff
@@ -15,6 +15,12 @@ public static void MyBuild()
         
         foreach (EditorBuildSettingsScene e in EditorBuildSettings.scenes)
         {
+            //Do not include AdditiveDebugScene in ProdBuild
+            if (e.path.Contains("AdditiveDebugScene"))
+            {
+                continue;
+            }
```

Comment:
Turns out it is easier just to remove from our Prod Build pipeline than manually add it to our Dev Build pipeline